README.md

```diff
@@ -1,6 +1,6 @@
 # @arcgis/core
```

Comment:
should this still be arcgis-js-api if it's the amd build?

Reply:
Good catch, fixed.